tests/test_files/plain_text/tables.txt

```diff
@@ -0,0 +1,5 @@
+Colons can be used to align columns.
```

Comment:
do you want text in here indicating that tables will be stripped from plain text?

Reply:
Not here. I documented that in the index file. The input markdown reflected here is copied directly from [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables).